lib/mixlib/install/backend/base.rb

```diff
@@ -123,14 +121,21 @@ def filter_artifacts(artifacts)
             return closest_compatible_artifact
           end
 
-           # Return an exception if we get to the end of the method
+          # Determine which platform version should be displayed in the exception
+          platform_version = if options.original_platform_version.nil?
+                               options.platform_version
+                             else
+                               options.original_platform_version
+                             end
```

Comment:
Related to https://github.com/chef/mixlib-install/pull/191/files#r106671866 we can skip this code if we just ensure `options.original_platform_version` is always set....we could then just do this below:

```ruby
platform version: #{options.original_platform_version}
```